src/components/Books.js

```diff
@@ -55,31 +77,43 @@ function CreateBook() {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
-      <div>
-        <label htmlFor="title">
-          Title:
-          <input
-            id="title"
-            type="text"
-            value={title}
-            onChange={(event) => setTitle(event.target.value)}
-          />
-        </label>
-      </div>
-      <div>
-        <label htmlFor="author">
-          Author:
-          <input
-            type="text"
-            id="author"
-            value={author}
-            onChange={(event) => setAuthor(event.target.value)}
-          />
-        </label>
+    <div className="add-book">
+      <hr width="100%" />
+      <h2>ADD NEW BOOK</h2>
+      <div className="add-book-items">
+        <form className="add-form" onSubmit={handleSubmit}>
+          <div>
+            <label htmlFor="title">
+              <input
+                id="title"
+                type="text"
+                value={title}
+                onChange={(event) => setTitle(event.target.value)}
+                placeholder="Book title"
+              />
+            </label>
+          </div>
+          {/* <div>
+            <label htmlFor="author">
+              <input
+                type="text"
+                id="author"
+                value={author}
+                onChange={(event) => setAuthor(event.target.value)}
+                placeholder="Author"
+              />
+            </label>
+          </div> */}
```

Comment:
- [ ] KIndly remove commented code as it makes your codebase not clean

Reply:
Removed! ✔️